android/build.gradle

```diff
@@ -54,9 +54,11 @@ android {
     }
 }
 
-project.apply {
-    from("$rootDir/gradle/ktlint.gradle")
-    from("$rootDir/gradle/lint.gradle")
+if (file("$rootDir/gradle/ktlint.gradle").exists() && file("$rootDir/gradle/lint.gradle").exists()) {
```

Comment:
is it possible to apply them independently?

Reply:
Not sure it's needed, linters are both needed for the development, but should be skipped for users